### DISCUSSION TITLE
Active crypto wallet key stealing repo

### DISCUSSION THREAD

**Discussion by ilikenwf** [community/community#147129] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I reported a Solana wallet private key stealer many days ago and have so far seen no action taken. The program in question has heavily obfuscated code that after many iterations I found sends stolen keys to a telegram channel to be drained.

Any and all action would be appreciated. There does appear to be forks of this program and the user account in question has other repos. I sent more pertinent information in my report, and have also reported to Telegram and the FBI.

https://github.com/h3curles/Pumpfun_SOLANA_SNIPER_BOT/blob/main/py_modules/netapp_api/config.py

**Comment by shinybrightstar** [comment]
Hi @ilikenwf , 👋🏻 We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam). Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!

**Comment by ghost** [comment]
https://github.com/h3curles/Pumpfun_SOLANA_SNIPER_BOT/blob/main/py_modules/netapp_api/config.py there is no repo

**Comment by ghost** [comment]
I hacked several time

**Comment by ghost** [comment]
As soon as I push private key in github, it is hacked

**Comment by ghost** [comment]
https://solscan.io/tx/4RjHKgBG9sFNU12nKxuoVTHXo8pUWYCSov5LungUDPFw9UPjHyYhVzTLDX7CzdMMPEeuuzJ5AA8kmYT2Nh9JfBU5
https://solscan.io/account/86etk2AXQmmsXXKE9YJTEUMb7GLYXZ7wNrCLoNWgdikt

**Comment by ghost** [comment]
can you explain this hacker

**Comment by ghost** [comment]
![image](https://github.com/user-attachments/assets/8db11143-36d7-4231-ae1d-e69999f61807)

**Comment by dylanw2486-oss** [comment]
Lost money to a crypto scam, romance scam, investment fraud, fake trading platform, phishing attack, or online fraud?

SAGITTARIUS CYPHER CYBER INTELLIGENCE is here to help you understand what happened and determine what steps may be available.

🌎 ASSISTANCE FOR CLIENTS WORLDWIDE

Our investigative and cyber-intelligence support covers:

🔎 Bitcoin & Cryptocurrency Scams
🔎 USDT & Ethereum Fraud
🔎 Romance & Relationship Scams
🔎 Investment & Trading Fraud
🔎 Fake Crypto Platforms & Trading Bots
🔎 Phishing & Impersonation
🔎 Online Shopping Scams
🔎 Digital-Asset Fraud
🔎 Cyber-Enabled Financial Fraud
🔎 Other Online & Offline Scams

WHAT WE CAN HELP WITH
We can help you organize and assess the information surrounding your case, identify relevant digital evidence, investigate available transaction information where appropriate, and understand potential reporting and recovery avenues.

You don't need to know what to do next. Start by telling us what happened.

⏱️ The sooner relevant evidence is preserved, the better.

🔐 Confidential Case Discussion
🌎 Worldwide Assistance
🔎 Professional Investigative Support

HAVE YOU BEEN SCAMMED?
Don't stay silent. Don't be embarrassed. And don't let another scammer convince you to pay for a guaranteed recovery.

Contact SAGITTARIUS CYPHER CYBER INTELLIGENCE today.

🌐 sagittariuscyphercyberintelligence.com

📧 info@sagittariuscyphercyberintelligence.com

📱 WhatsApp: +1 (226) 457-3805

💬 Telegram: @sagittariuscyphercyber

Tell us what happened. We'll help you understand your options.

SAGITTARIUS CYPHER CYBER INTELLIGENCE does not guarantee  but give you 100% recovery of lost funds or assets. Each case is assessed individually based on the circumstances and available evidence.
